train.py

```diff
@@ -98,7 +100,14 @@
             args.tensorboard_log = ''
 
     for env_id in env_ids:
-        tensorboard_log = None if args.tensorboard_log == '' else os.path.join(args.tensorboard_log, env_id)
+        log_path = "{}/{}/".format(args.log_folder, args.algo)
```

Comment:
maybe it is time to replace that line with `os.path.join` so it works on windows too?